rails-task-list/app/controllers/tasks_controller.rb

```diff
@@ -30,19 +30,24 @@ def edit
 
   def update
     id = params[:id]
-    Task.find(id).update(task_params[:task])
+    #Task.find(id).update(task_params[:task])
+    Task.update(id, task_params[:task])
     redirect_to "/"
   end
 
   def complete
     id = params[:id]
-    Task.find(id).update(:completed_date => Time.now)
+    task = Task.find(id)
+    task.update(:completed_date => Time.now)
+    num_tasks = task.person.lifetime_tasks
+    num_tasks += 1
+    task.person.update(:lifetime_tasks => num_tasks)
```

Comment:
I think know if it is a good idea to keep a separate column/attribute to track the number of tasks. I think it would be better to calculate this dynamically whenever its needed as opposed to storing it in the database